apps/postybirb-ui/src/app/app-i18n-provider.tsx

```diff
@@ -22,29 +30,61 @@ export function AppI18nProvider(props: AppI18nProviderProps) {
       // eslint-disable-next-line no-param-reassign
       locale = locale ?? 'en';
       const { messages } = await import(`../lang/${locale}.po`);
+
+      const uppyLocale = uppyLocales[locale];
+      try {
+        i18n.uppy = (
+          await import(`../../public/uppy-i18n/${uppyLocale}.js`)
+        ).default;
+      } catch (error) {
+        // eslint-disable-next-line no-console
+        console.error(
+          // eslint-disable-next-line lingui/no-unlocalized-strings
+          `Failed to load uppy locale for ${locale}, mapped: ${uppyLocale}, error:`,
+          error
+        );
```

Comment:
instead of not loading i18n for the whole application, show helpfull error message to the console